docs/docs/features/studio.md

```diff
@@ -10,6 +10,34 @@ ZMK Studio is in beta. Although every effort has been made to provide a stable e
 
 ZMK Studio provides runtime update functionality to ZMK powered devices, allowing users to change their keymap layers without flashing new firmware to their keyboards. Studio is still under active development, and is not yet ready for casual end user use.
 
+## Capabilities
+
+ZMK Studio currently has the following capabilities:
+
+| Legend: | ✅ Supported | 🚧 Under Development | 💡 Planned | ❌ Not Planned | ? Pete, help pls |
+| :------ | :----------- | :------------------- | :--------- | :------------- | ---------------- |
+
+| Studio Status | **Feature or Capability**                                                                         |
+| :-----------: | ------------------------------------------------------------------------------------------------- |
+|      ✅       | Making keymap changes while keyboard is in use                                                    |
+|      ✅       | Making changes while connected via USB                                                            |
+|      ✅       | Native app for: Windows, Linux, MacOS                                                             |
+|      ✅       | Making changes while connected via BLE (Linux web-app & native apps only)                         |
+|      ✅       | Assigning [predefined behaviors](../keymaps/behaviors/index.mdx) to keys                          |
+|      ✅       | Assigning [user-defined behaviors](../keymaps/behaviors/index.mdx#user-defined-behaviors) to keys |
+|      💡       | Defining new [user-defined behaviors](../keymaps/behaviors/index.mdx#user-defined-behaviors)      |
```

Comment:
Not really... Sorta, but not really? We'll have support for "reserved" behaviors of a given type, much like we have reserved keymap layers, that can then be "added" and further configured from there.